templates/data/summary.html

```diff
@@ -24,25 +31,38 @@
           <tr>
             <td>artist</td>
             <td>
-              {% if lowlevel.metadata.tags.musicbrainz_artistid %}
-                <a href="https://musicbrainz.org/artist/{{ lowlevel.metadata.tags.musicbrainz_artistid[0] }}">
-                  {{ lowlevel.metadata.tags.artist[0] }}
-                </a>
-              {% else %}
-                {{ lowlevel.metadata.tags.artist[0] }}
-              {% endif %}
+              {%- if metadata -%}
+                {# TODO(roman): Handle multiple artists properly. #}
+                {%- set artist_id = metadata['artist-credit'][0]['artist']['id'] -%}
+                <a href="https://musicbrainz.org/artist/{{ artist_id }}">{{ metadata['artist-credit-phrase'] }}</a>
+              {%- else -%}
+                {%- if lowlevel.metadata.tags.musicbrainz_artistid -%}
```

Comment:
Is it worth verifying if the tag information (if present) is the same or different from musicbrainz information? Maybe it could be an interesting report to see "badly tagged" file? On the other hand, maybe it would get noisy as stuff on MB is edited for normal reasons